webnn/concat.https.any.js

```diff
@@ -0,0 +1,72 @@
+// META: title=test WebNN API concat operation
+// META: global=window,dedicatedworker
+// META: script=./resources/utils.js
+// META: timeout=long
+
+'use strict';
+
+// https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-concat
+
+const testConcat = async (operandType, syncFlag, inputShapeValues, axis, expectedShapeValue) => {
+  const inputs = [];
+  const namedInputs = {};
+  const TestTypedArray = TypedArrayDict[operandType];
+  for (let i = 0; i < inputShapeValues.length; i++) {
+    inputs.push(builder.input('input' + i, {type: operandType, dimensions: inputShapeValues[i].shape}));
+    namedInputs['input' + i] = new TestTypedArray(inputShapeValues[i].data);
+  }
+  const outputOperand = builder.concat(inputs, axis);
+  const outputs = {'outputOperand': new TestTypedArray(sizeOfShape(expectedShapeValue.shape))};
+  let graph;
+  if (syncFlag) {
+    graph = builder.buildSync({outputOperand});
+    context.computeSync(graph, namedInputs, outputs);
+  } else {
+    graph = await builder.build({outputOperand});
+    await context.compute(graph, namedInputs, outputs);
+  }
```

Comment:
Is that possible to wrap this code snippet into a common function? Which could be shared to other tests.
````
  let graph;
  if (syncFlag) {
    graph = builder.buildSync({outputOperand});
    context.computeSync(graph, namedInputs, outputs);
  } else {
    graph = await builder.build({outputOperand});
    await context.compute(graph, namedInputs, outputs);
  }
```

Reply:
I extracted them into `buildAndCompute()` function of `utils.js`, and also did for the codes of creating context and builder by `createContextAndBuilder()` function, thanks.